mathics/builtin/calculus.py

```diff
@@ -516,6 +516,46 @@ def apply(self, f, xs, evaluation):
         return result
 
 
+class Root(Builtin):
+    """
+    <dl>
```

Comment:
are these xml tags correct? there is maybe some missing closing tags?

Reply:
actually i just compared it, seems fine

Reply:
Yeah, I was basing this in the other docs.